<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerMonitor</class>
 <widget class="QMainWindow" name="PowerMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sp_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set Point</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sp_doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <double>0.800000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphwidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>871</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dt_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Interval(ms)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dt_SpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="inf_checkBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="win_SpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="win_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QLabel" name="inf_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Infinite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
